.golangci.yml

```diff
@@ -1,11 +1,10 @@
 run:
-  timeout: 5m
+  modules-download-mode: readonly
+
 output:
-  # format: checkstyle:golangci-report.xml
   print-issued-lines: false
-  # print-linter-name: false
   sort-results: true
-# All possible options can be found here https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml
+
 linters-settings:
```

Comment:
Could we test if the following setting adds a lot of false positives? I think it's a good addition based on our reviews:
```yaml
linters-settings:
  unparam:
    check-exported: true
```